collections/_posts/_projects/meshery.md

```diff
@@ -1,10 +0,0 @@
----
```

Comment:
Is the relocation of this file intentional?

Reply:
no